docs/docs/examples-tutorials/recipes/running-tracetest-with-dynatrace.md

```diff
@@ -0,0 +1,356 @@
+
+To start both the OpenTelemetry Demo and Tracetest we will run this command:
+
+```bash
+docker-compose -f docker-compose.yaml -f tracetest/docker-compose.yaml up
+```
+
+The `tracetest-config.yaml` file contains the basic setup of connecting Tracetest to the Postgres instance and telemetry exporter. The exporter is set to the OpenTelemetry Collector.
+
+```yaml
+# tracetest-config.yaml
+
+---
+postgres:
+  host: postgres
+  user: postgres
+  password: postgres
+  port: 5432
+  dbname: postgres
+  params: sslmode=disable
+
+telemetry:
+  exporters:
+    collector:
+      serviceName: tracetest
+      sampling: 100
+      exporter:
+        type: collector
+        collector:
+          endpoint: otel-collector:4317
+
+server:
+  telemetry:
+    exporter: collector
+    applicationExporter: collector
+```
+
+The `tracetest-provision.yaml` file contains the setup of the demo APIs that Tracetest can use as an example for tests, the polling profiles that say how Tracetest should fetch traces from the data store and the configuration for the data store, in our case, Datadog.
+
+```yaml
+---
+type: Demo
+spec:
+  name: "OpenTelemetry Shop"
+  enabled: true
+  type: otelstore
+  opentelemetryStore:
+    frontendEndpoint: http://frontend:8084
+    productCatalogEndpoint: productcatalogservice:3550
+    cartEndpoint: cartservice:7070
+    checkoutEndpoint: checkoutservice:5050
+
+---
+type: PollingProfile
+spec:
+  name: Default
+  strategy: periodic
+  default: true
+  periodic:
+    retryDelay: 5s
+    timeout: 180s
+
+---
+type: DataStore
+spec:
+  name: datadog
+  type: datadog
+
+```
+
+**Sending Traces to Tracetest and Datadog**
+
+The `collector.config.yaml` explains that. It receives traces via either `grpc` or `http`. Then, exports them to Tracetest's OTLP endpoint `tracetest:4317` in one pipeline, and to Datadog in another.
+
+Make sure to add your Datadog API Key to the `datadog` exporter.
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      http:
+      grpc:
+
+processors:
+  batch: # this configuration is needed to guarantee that the data is sent correctly to Datadog
+    send_batch_max_size: 100
+    send_batch_size: 10
+    timeout: 10s
+
+exporters:
+  # OTLP for Tracetest
+  otlp/tracetest:
+    endpoint: tracetest:4317
+    # Send traces to Tracetest.
+    # Read more in docs here: https://docs.tracetest.io/configuration/connecting-to-data-stores/opentelemetry-collector
+    tls:
+      insecure: true
+  # Datadog exporter
+  datadog:
+    api:
+      site: datadoghq.com
+      key: ${DATADOG_API_KEY} # Add here you API key for Datadog
+    # One example on how to set up a collector configuration for Datadog can be seen here:
+    # https://docs.datadoghq.com/opentelemetry/otel_collector_datadog_exporter/?tab=onahost
```

Comment:
Here we need to change to Dynatrace-specific configurations, probably similar to the setup here: `examples/tracetest-dynatrace/tracetest/collector.config.yaml`